monai/data/wsi_reader.py

```diff
@@ -174,7 +174,7 @@ def get_data(
             # Verify location
             if location is None:
                 location = (0, 0)
-            wsi_size = self.get_size(each_wsi, level)
+            wsi_size = self.get_size(each_wsi, 0)
```

Comment:
May I know why change `level` to `0` here?

Thanks.

Reply:
It was a bug that is fixed. For `wsi_size`, I need the size of the full resolution image, which is level `0`.